src/olympia/devhub/templates/devhub/addons/submit/source.html

```diff
@@ -63,6 +63,11 @@ <h3>{{ _("You must submit your extension's source code.") }}</h3>
                 Source files (aside from open-source third party libraries) cannot be transpiled, concatenated, minified, or otherwise machine-generated.
             {% endtrans %}
         </p>
+        <p>
+            {% trans %}
+            The maximum file size accepted is 200MB.
```

Comment:
Is there a setting or some constant determining what the actual max file size is? I would recommend a) linking to that in an html comment or b) (better but more work) using that value in the template so they are always synchronized.

Reply:
The one attributed to the error shown to the user, as far as I could tell, was just a hard-coded variable in the JS (upload-addon.js#38 used in upload-base.js#83). I added a comment next to it

Reply:
([This](https://github.com/mozilla/addons/issues/15039#issuecomment-2368479933) would change things, though)